readme.md

```diff
@@ -57,7 +57,7 @@ ___
 ## Prerequisites
 
 - [NodeJS (18.x.x)](https://nodejs.org/en/) 
-- [.NET Core SDK (5.0.100)](https://dotnet.microsoft.com/download/dotnet-core/5.0)
+- [.NET Core SDK (7.0.404)](https://dotnet.microsoft.com/en-us/download/dotnet/7.0)
```

Comment:
remove `en-us` from url and indicate  `or above`  https://github.com/Azure/autorest.csharp/pull/3941#discussion_r1398642266